superchain/init.go

```diff
@@ -105,3 +108,9 @@ func init() {
 		Implementations[superchainEntry.Config.L1.ChainID] = implementations
 	}
 }
+
+func checkSuperchainLevel(chainConfig ChainConfig) {
+	if chainConfig.SuperchainLevel != Frontier && chainConfig.SuperchainLevel != Standard {
+		panic(fmt.Sprintf("invalid or unspecified superchain level %d", chainConfig.SuperchainLevel))
```

Comment:
There's not a super clear way to log errors during init except for panics. It's an ok usage. The other option is to add a test for this.

Reply:
We could migrate it into a test, I put it here because I felt it is more about data integrity. Be most happy to follow up in a later PR, mostly just keen to make sure in flight PRs don't forget to set this.

Reply:
Could we rename this to something like `MustBeValidSuperchainLevel` because `Must` functions are assumed to be able to panic? Then I'm happy with these changes

Reply:
[da2253c](https://github.com/ethereum-optimism/superchain-registry/pull/167/commits/da2253c181e17a4d45379ec80534f2fead612c75)